docs/6.x/installation.md

```diff
@@ -135,14 +135,37 @@ Flag               | Description
 `--service-uid`    | _(Optional)_ Service user ID (numeric). See [Service User](pack/#service-user) for details. A user named `planet` is created automatically if unspecified.
 `--service-gid`    | _(Optional)_ Service group ID (numeric). See [Service User](pack/#service-user) for details. A group named `planet` is created automatically if unspecified.
 
+### Environment Variables
+
+Some aspects of the installation can be configured with the use of environment variables.
+
+#### GRAVITY_CHECKS_OFF
+
+This environment variable controls whether the installer/join agent executes the preflight checks.
+It accepts values `1`, `t`, `T`, `TRUE`, `true`, `True`, `0`, `f`, `F`, `FALSE`, `false`, `False`
+- any other value will not have any effect.
+
+!!! warning "Scope":
+    It is not possible to selectively turn checks off - it is all or nothing.
```

Comment:
Maybe add "**Currently** it is not possible..." because we have this ability planned.